### ISSUE TITLE
[Issue]: Can not choose language or version on videos while using SyncPlay

### ISSUE THREAD

**Issue by DevPGSV** [jellyfin/jellyfin#9142]
### Please describe your bug

While using SyncPlay with someone else, it is not possible to change the language of a movie/tv show to anything other than the default.

![image](https://user-images.githubusercontent.com/5631093/213814291-9a33b67d-b94d-4ac3-bf7b-9a25f3f2846e.png)
![image](https://user-images.githubusercontent.com/5631093/213814356-b7973490-abc8-4ecc-a70a-445527ff2462.png)

While on a SyncPlay group, watching a movie with an audio stream different than the default, as soon as someone else joins the group to start watching the same movie, the audio stream is changed to the default one. While someone else is on the same SyncPlay group, changing the audio stream has no effect.

Also, if that movie has multiple versions:
![image](https://user-images.githubusercontent.com/5631093/213814802-dad82caf-6ba5-40f4-bd83-e6b4cd99ea52.png)
While using SyncPlay, the version used is also changed to the default one, without any way to choose the version of the movie.

### Jellyfin Version

Other

### if other:

10.8.7

### Environment

```markdown
- OS: ArchLinux
- Virtualization: Docker
- Clients: Browser
- Browser: Brave Version 1.46.153 Chromium: 108.0.5359.128 (Build oficial) (64 bits)
```

### Jellyfin logs

```shell
No logs relevant to this issue
```

### FFmpeg logs

```shell
No logs relevant to this issue
```

### Please attach any browser or client logs here

No logs relevant to this issue

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Gorkycreator** [comment]
I encountered this problem yesterday. The movie we were trying to watch had a 4k and an HD version, correctly labeled and in working order outside the syncplay group. The other user we were watching with experienced choppy playback and I noticed the playback options reported that the 4k version was playing (despite the 1080p version appearing to be selected on the video details menu before we clicked "play"). Attempting to restart the movie with different settings within the current syncplay session failed to alter which version Jellyfin started playing, it always reported the 4k version was running in the playback info.

We tried restarting the syncplay session a few times, alternating who created and drove the group, and eventually got it to play the HD version, but accidentally with an voice-over audio track that was included in the MKV file. Attempts to select a different track were unsuccessful.

Ultimately, I just deleted the 4k version of the film to work around the problem.
